tensorflow_serving/tools/docker/Dockerfile.mkl

```diff
@@ -81,11 +81,23 @@ ENV KMP_SETTINGS=1
 ENV KMP_AFFINITY='granularity=fine,verbose,compact,1,0'
 ENV MKLDNN_VERBOSE=0
 
-ENV TENSORFLOW_SESSION_PARALLELISM=2
+# You can set either TENSORFLOW_SESSION_PARALLELISM or
```

Comment:
Is there any reason why someone would want to keep session parallelism? Given they have to modify the dockerfile regardless (and that there are a ton of flags not pre-configured for a user), it's not clear we need to bother with this.

Reply:
We don't need this be configured any more. Just for user to know i added those comments. But can remove them.

Reply:
You want me to get rid off that ?

Reply:
Yeah, I'd remove them in that case. Makes it cleaner/easier to understand.

Reply:
Sure will update the changes.